app/controllers/notifications_controller.rb

```diff
@@ -5,7 +5,7 @@ class NotificationsController < ApplicationController
 
   # GET /notifications
   def index
-    @notifications = Notification.where(recipient_id: current_member).page(params[:page])
+    @notifications = Notification.find_by_recipient(current_member).page(params[:page])
```

Comment:
Might be better with the 'by_recipient' instead of find_by_recipient, as it looks like the normal find_by method that would only return one record; not a .where